content/en/docs/components/pipelines/sdk-v2/importer-component.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Importer component"
```

Comment:
Maybe hold off on this doc for now. With the upcoming pipeline input artifact support, I personally had a hard time to justify if importer would still be needed. Maybe there're cases I haven't thought about where importer is still useful. Happy to chat more offline.

Reply:
Synced offline, we can merge this when ready given importer is already exposed to existing users, this just completes the documentation part. We can discuss the usage of importer later.